programs/marinade-finance/src/instructions/user/withdraw_stake_account.rs

```diff
@@ -175,6 +183,7 @@ impl<'info> WithdrawStakeAccount<'info> {
             // the fee value will be burned but not delivered, thus increasing mSOL value slightly for all mSOL holders
```

Comment:
This comment should be adjusted as instead of burning there will be moving to treasury.